lib/client/weblogin.go

```diff
@@ -364,6 +385,78 @@ func SSHAgentLogin(ctx context.Context, login SSHLoginDirect) (*auth.SSHLoginRes
 	return out, nil
 }
 
+// SSHAgentPwdlessLogin requests a passwordless MFA challenge via the proxy.
+// If the credentials are valid, the proxy will return a challenge. We then
+// prompt the user to touch a device and if user wasn't explicitly defined,
+// also prompts the user to select a login name. If the authentication succeeds,
+// we will get a temporary certificate back.
```

Comment:
nit: Drop the "we"s, call out wancli.LoginPrompt and simplify?

I wouldn't spell out the prompt flow here, it has many variations.

```suggestion
// login.CustomPrompt (or a default prompt) is used if for interaction with the
// end user.
// Returns the SSH certificate it authn is successful or an error.
```